include/picongpu/particles/atomicPhysics/kernel/FillLocalRateCache_BoundBound.kernel

```diff
@@ -154,7 +154,8 @@ namespace picongpu::particles::atomicPhysics::kernel
 
             auto forEachAtomicStateAndBin
                 = pmacc::lockstep::makeForEach<T_numberAtomicStates * T_numberBins, T_Worker>(worker);
-            auto forEachAtomicState = pmacc::lockstep::makeForEach<T_numberAtomicStates, T_Worker>(worker);
+            [[maybe_unused]] auto forEachAtomicState
+                = pmacc::lockstep::makeForEach<T_numberAtomicStates, T_Worker>(worker);
```

Comment:
```suggestion
            auto forEachAtomicState
                = pmacc::lockstep::makeForEach<T_numberAtomicStates, T_Worker>(worker);
```
and move it down to where it's actually used. As far as I can see that's a single place. And maybe do the same for the `forEachAtomicStateAndBin` above.

Reply:
it is used twice in two `if constexpr` branches but would like avoid creating the for each twice.

Reply:
Doing "Click `...`" -> "View File" -> "Ctrl-F" -> "`forEachAtomicState`" -> "Enter" yields exactly four results for me: One definition and one usage per variable. Are you aware that these are two different variables?

Reply:
I missed it, the names are so equal that I oversaw it. 

fixed it